docs/Compile.rst

```diff
@@ -121,7 +156,9 @@ DF, which causes DF to use your system libstdc++ instead::
 Note that distributing binaries compiled with newer GCC versions requires end-
 users to delete libstdc++ themselves and have a libstdc++ on their system from
 the same GCC version or newer. For this reason, distributing anything compiled
-with GCC versions newer than 4.8 is discouraged.
```

Comment:
(Of course, the system GCC is fine for personal builds as long as you get rid of DF's libstdc++.)